<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>197</y>
     <width>539</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QComboBox" name="cmbEcho">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>54</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkReadOnly">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadOnly</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>106</y>
      <width>57</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>192</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtDefault"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>109</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtTest">
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
